### ISSUE TITLE
`parse_commands` calls in `server.py` doesn't pass the `cfg` parameter.

### ISSUE THREAD

**Issue by kennytm** [servo/homu#154]
In the following places, the `parse_commands()` call doesn't pass the `cfg` parameter, 

https://github.com/servo/homu/blob/e4b2fc0a5e376c6c87e3d21c0b1a5fb834250db5/homu/server.py#L302-L312

https://github.com/servo/homu/blob/e4b2fc0a5e376c6c87e3d21c0b1a5fb834250db5/homu/server.py#L348-L356

... so commands like r+/r- are failing with

```
Traceback (most recent call last):
  File "/home/kennytm/homu/env/bin/bottle.py", line 862, in _handle
    return route.call(**args)
  File "/home/kennytm/homu/env/bin/bottle.py", line 1740, in wrapper
    rv = callback(*a, **ka)
  File "/home/kennytm/homu/homu/server.py", line 450, in github
    realtime=True,
TypeError: parse_commands() missing 1 required positional argument: 'states'
```

cc @alexrs #152

**Comment by alexrs** [comment]
@kennytm Thanks for notifying it and sorry for the inconvenience. We are refactoring the code to add tests to avoid these problems, but we don't have enough tests yet to detect all the possible issues. I just created https://github.com/servo/homu/pull/155 that should fix the issue.

If you find any other problem, don't hesitate to report it!

**Comment by kennytm** [comment]
(Fixed in #155, closing)
